<ui version="4.0" >
 <class>AddNewClassVariable</class>
 <widget class="QDialog" name="AddNewClassVariable" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add New Variable</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Create get/set methods</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <widget class="QCheckBox" name="setInline" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>inline</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QCheckBox" name="getInline" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>inline</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="setName" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="set" >
        <property name="text" >
         <string>set method</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="getName" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="get" >
        <property name="text" >
         <string>get method</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>340</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Variable Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Scope:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="variableName" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="type" >
           <property name="editable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="scope" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="editable" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>variableName</tabstop>
  <tabstop>type</tabstop>
  <tabstop>scope</tabstop>
  <tabstop>get</tabstop>
  <tabstop>getName</tabstop>
  <tabstop>getInline</tabstop>
  <tabstop>set</tabstop>
  <tabstop>setName</tabstop>
  <tabstop>setInline</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddNewClassVariable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>347</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel" >
     <x>325</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>get</sender>
   <signal>toggled(bool)</signal>
   <receiver>getName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>65</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel" >
     <x>148</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>get</sender>
   <signal>toggled(bool)</signal>
   <receiver>getInline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>77</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel" >
     <x>293</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set</sender>
   <signal>toggled(bool)</signal>
   <receiver>setName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel" >
     <x>145</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set</sender>
   <signal>toggled(bool)</signal>
   <receiver>setInline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>23</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel" >
     <x>315</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
